app/uk/gov/hmrc/personaldetailsvalidation/views/pages/incorrect_details_accordion.scala.html

```diff
@@ -71,10 +71,11 @@
         "multi_option_incorrect_details.accordion.accordion3.6",
         "multi_option_incorrect_details.accordion.accordion3.7",
         "multi_option_incorrect_details.accordion.accordion3.8",
-        "multi_option_incorrect_details.accordion.accordion3.9"
+        "multi_option_incorrect_details.accordion.accordion3.9",
+        "multi_option_incorrect_details.accordion.accordion3.10"
     ))
-    @p(msgs("multi_option_incorrect_details.accordion.accordion3.10","https://www.gov.uk/lost-national-insurance-number"))
     @p(msgs("multi_option_incorrect_details.accordion.accordion3.11","https://www.gov.uk/apply-national-insurance-number"))
```

Comment:
Above this line of text in the screenshot there appears to be a heading missing:

I do not have a National Insurance number, or have lost or forgotten it